src/ansible/roles/ssh_server/tasks/main.yml

```diff
@@ -11,3 +11,7 @@
     name: '{{ systemd.services.sshd }}'
     enabled: yes
     state: restarted
+  register: restart_res
+  until: "restart_res is not failed"
```

Comment:
Please, add comment why restart is needed.

Reply:
Done.